### DISCUSSION TITLE
应用a如何监听应用b的配置变化，类似@ApolloConfigChangeListener

### DISCUSSION THREAD

**Discussion by BigWrite** [apolloconfig/apollo#3846] [Q&A]
如题，springboot+apollo，现在想在应用a中实时监听应用b的配置变化（对应apollo就是不同的appId，namespace是私有的），但是发现@ApolloConfigChangeListener只支持监听当前config的，有什么好的方法吗

**Comment by Anilople** [comment]
什么场景下有这个需求？

如果应用a需要监听应用b的配置，那么这个配置能否变成一个公共配置？这样a和b都可以使用，并且监听变化

**Reply by BigWrite** [reply]
应用a 可以获得 应用b的config 然后使用注解吗，应用b的配置是私密的，不太好改成公共配置，之前是通过openApi 全量更新检测的，现在想只检测变化数据

**Reply by BigWrite** [reply]
如果要切换配置的类型，怎么切换呢，没找到对应的功能在哪😂

**Reply by nobodyiam** [reply]
可以新建一个公共 namespace
